src/bond/james.cljc

```diff
@@ -20,12 +20,14 @@
 (defn calls
   "Takes one arg, a fn that has previously been spied. Returns a seq
   of maps, one per call. Each map contains the keys :args and :return
-  or :throw."
+  or :throw. If the fn has not been spied, throws an exception."
```

Comment:
Should we bump the release version to 0.5.0 or 1.0.0 to indicate the breaking change?

Reply:
First, let's decide if this constitutes a breaking change. On the one hand, the behavior of using `calls` on an unspied fn was not specified by the docstring. On the other hand, this change would cause any tests that relied on the old behavior to fail. But, this change could cause tests to fail that should have been failing: suppose a developer upgrades bond, sees a test failure because `calls` was used on an unspied fn, modifies the test to spy the fn, and then sees another test failure because fn was called more than once when it should have been called 0 times.

What do you think? Should this constitute a breaking change?

I think we should wait to merge this until we find evidence that users rely on the old behavior, but that behavior obscures a bug. If we find evidence supporting this, then the proposed behavior of `calls` is useful, and I think we should release it as 0.5.0. If we don't, then I think we should scrap the proposed behavior.

Reply:
Agreed, "breaking" might be too strong. I definitely think it's worth merging because the existing behaviour is surprising and that the change in behaviour is worth surfacing in the version number. I'm good with releasing it as 0.5.0 👍 

Speaking of which, it looks like this has been released manually in the past, because I can't see anything in the CI 
or project config. Let me know if you'd like a hand with it.

Reply:
I have tried this PR out in a large test suite that uses bond. My modified bond found a small number of assertions about functions that were not spied, and all these assertions were easily fixed. So in practice, this PR is not really breaking, and the new behavior should help avoid pitfalls.

Given this, let's merge this and release 0.5.0. I would appreciate help with making the release :)